### PR TITLE
Add max affected version back into Note Detail.

### PR DESCRIPTION
**Pull request by nms2140**

### CODE REVIEW

File: proto/v1/vulnerability.proto [hunk 1]
```diff
@@ -73,8 +73,16 @@ message VulnerabilityNote {
...
+    // Required. The starting_affected_version is the starting version number of
+    // one of thoes intervals.
+    grafeas.v1.Version starting_affected_version = 6;
+
+    // Required. The ending version of an interval in which this vulnerability
+    // exists.
+    grafeas.v1.Version ending_affected_version = 11;
 
     // The [CPE URI](https://cpe.mitre.org/specification/) this vulnerability
     // was fixed in. It is possible for this to be different from the
```
Comment (vtsao) [inline proto/v1/vulnerability.proto]: fixed_cpe_uri:
The distro recommended CPE to update to that contains a fix for this vulnerability.

fixed_package:
The distro recommended package to update to that contains a fix for this vulnerability.

fixed_version:
The distro recommended version to update to that contains a fix for this vulnerability.

File: proto/v1/vulnerability.proto [hunk 2]
```diff
@@ -73,8 +73,16 @@ message VulnerabilityNote {
     // Required. The package this vulnerability affects.
```
Comment (vtsao) [inline proto/v1/vulnerability.proto]: You'll need to update the validators and tests as well.

File: Makefile [hunk 3]
```diff
@@ -17,13 +17,8 @@ default: .check_makefile_in_gopath build
 	@touch $@
 
 EXPECTED_MAKE = ${GOPATH}/src/github.com/grafeas/grafeas/Makefile
-
 .check_makefile_in_gopath:
```
Comment (vtsao) [inline Makefile]: Revert?

**Comment by vtsao** [comment]
Want LGTM from @aysylu as well.

File: proto/v1/vulnerability.proto [hunk 4]
```diff
@@ -73,8 +73,16 @@ message VulnerabilityNote {
...
     string affected_package = 5;
 
-    // Required. The minimum version of the package this vulnerability affects.
-    grafeas.v1.Version min_affected_version = 6;
+    // A vulnerability can affect a package between version numbers that
+    // are disjoint sets of intervals (Example: [1.0.0-1.1.0], [2.4.6-2.4.8] and 
+    // [4.5.6-4.6.8].
```
[the remark this answers is not in the document]
Comment (nms2140) [reply proto/v1/vulnerability.proto]: Done.

File: proto/v1/vulnerability.proto (hunk 1 above)
Comment (nms2140) [reply proto/v1/vulnerability.proto]: Done.

File: Makefile (hunk 3 above)
Comment (nms2140) [reply Makefile]: Done.

File: proto/v1/vulnerability.proto [hunk 5]
```diff
@@ -73,8 +73,16 @@ message VulnerabilityNote {
...
+    // A vulnerability can affect a package between version numbers that
+    // are disjoint sets of intervals (Example: [1.0.0-1.1.0], [2.4.6-2.4.8] and 
+    // [4.5.6-4.6.8]. 
+    // Required. The starting_affected_version is the starting version number of
+    // one of thoes intervals.
+    grafeas.v1.Version starting_affected_version = 6;
+
+    // Required. The ending version of an interval in which this vulnerability
+    // exists.
+    grafeas.v1.Version ending_affected_version = 11;
```
[the remark this answers is not in the document]
Comment (nms2140) [reply proto/v1/vulnerability.proto]: Done.

**Comment by vtsao** [comment]
Merging so @nms2140 can move forward. @aysylu if you have more comments we can address in a follow-up PR.

**Comment by aysylu** [comment]
LGTM, thank you.

**Pull request merged**